lib/librarian/ansible/source/galaxy.rb

```diff
@@ -58,13 +58,13 @@ def lookup_package(username, name)
                 break if json['results'].nil?
 
                 package = json['results'].find do |r|
-                  r['namespace'] == username &&
-                    r['name'] == name
+                  r['summary_fields']['namespace']['name'] == username &&
+                      r['name'] == name
```

Comment:
It doesn’t matter much, but this line is indented one level too deep.

Reply:
I was just appliying the formatting rules from my intellij. But i can fix that